actions/utils.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/**
```

Comment:
Could we rename this file to AuthUtils (more specific) and also use PascalCase?
Or, if this code isn't used anywhere else, I'd prefer to keep it in the Auth Action.

Reply:
>Or, if this code isn't used anywhere else, I'd prefer to keep it in the Auth Action.

Right now it is used only in the Auth Action. I moved it there.